dex/multi_token_farm/src/lib.rs

```diff
@@ -7,10 +7,7 @@ mod rewards;
 
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
-use dex_common::{FftTokenAmountPair, GenericEsdtAmountPair};
-
-type Epoch = u64;
-type Nonce = u64;
+use common_structs::{Epoch, FftTokenAmountPair, GenericTokenAmountPair, Nonce};
```

Comment:
Just to make sure: this file is deprecated and it is not under the scope of review

Reply:
Yes. No review needed.